core/src/avm2/globals/README.md

```diff
@@ -10,17 +10,15 @@ Adobe's `playerglobal.swf` uses native methods that Ruffle doesn't implement).
 
 Currently, globals are implemented in one of two ways:
 1) As pure-Rust code in files like `system.rs`. These are normal Rust
-   modules, and are used from `core/src/avm2/global.rs`
+   modules, and are used from `core/src/avm2/global.rs`.
 2) As ActionScript class definitions like `flash/geom/Point.as`.
-   These files are automatically compiled into a `playerglobal.swf`
-   file at build time, which is included into the final Ruffle binary
+   The files included from `globals.as` are compiled into a `playerglobal.swf`
+   file at build time, which is shipped with the final Ruffle binary
```

Comment:
I think it would good  to emphasize that this is part of the final binary, and not a separate file that gets distributed with it.